pkg/traceql/ast.go

```diff
@@ -722,7 +722,7 @@ func (Attribute) referencesSpan() bool {
 func NewScopedAttribute(scope AttributeScope, parent bool, att string) Attribute {
 	intrinsic := IntrinsicNone
 	// if we are explicitly passed a resource or span scopes then we shouldn't parse for intrinsic
-	if scope != AttributeScopeResource && scope != AttributeScopeSpan {
+	if scope == AttributeScopeNone && !parent {
```

Comment:
currently
```
if scope != AttributeScopeResource && scope != AttributeScopeSpan
```
this would allow {parent.duration} to parse to IntrinsicDuration and we discussed to make this to parse it as "duration" as an attribute